apis/hive/v1/hiveconfig_types.go

```diff
@@ -12,6 +12,10 @@ const (
 
 	// FeatureGateMachineManagement enables the use of the central machine management alpha.
 	FeatureGateMachineManagement = "AlphaMachineManagement"
+
+	// FeatureGateClusterInstallReference enables the use of hivecontract ClusterInstall reference
+	// in ClusterDeployment.
+	FeatureGateClusterInstallReference = "AlphaClusterInstallReference"
```

Comment:
Good thinking.

Reply:
OTOH Assisted teams and ACM are both gearing up for AlphaAgentInstallStrategy, it might be worth doing the wrong thing and just assuming that AlphaAgentInstallStrategy means to enable the new CRDs and functionality, as it's only for 4.8 regardless. This would limit churn for them and us trying to coordinate and communicate the change. WDYT?

Reply:
sure i can re use the same feature gate.